server.py

```
@@ -99,6 +100,31 @@ def deduplicate_svg_attrs(svg_string: str) -> str:
 PIECE_SETS = svg.available_piece_sets()
+DEFAULT_PIECE_SET = "cburnett"
+
+
+def query_bool(request, name, default=False):
+    value = request.query.get(name)
+    if value is None:
+        return default
+    try:
+        return configparser.ConfigParser.BOOLEAN_STATES[value.lower()]
+    except KeyError:
+        raise aiohttp.web.HTTPBadRequest(reason=f"{name} must be a boolean")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)